### PR TITLE
Changes and updates

### PR DESCRIPTION
**Pull request by aalavandhaann**
# Changes in `tikz_export.py`
- The node tag for image that is written has been changed internally in the code inside ``_handle_image``. 
- Previously the whole image tag was written inside `_write_tikz_path` and this was not following the idea of the author, which is more sensible. 
- Adding the option of `scale` in the argument parser outside the `if not self.inkscape_mode:` in the method `_set_up_options`. This is to facilitate the ability to give a scale value in the Inkscape export settings.
- The GUI for export settings has been updated (file `tikz_export_effect.inx`) reflecting the changes made for scale value to be controlled by the user
- And a wiki section has been added with appropriate screenshots added to the repository. Hence README.md has also been updated.

### CODE REVIEW

**Comment by kjellmf** [comment]
Thanks. I will review the PR soon.

File: svg2tikz/extensions/tikz_export.py [hunk 1]
```diff
@@ -1040,16 +1040,15 @@ def _handle_image(self, node):
         
         width =  self.pxToPt(self.unittouu(node.get('width', '0')));
         height = self.pxToPt(self.unittouu(node.get('height', '0')));
-        
+#         opacity = float(dict(map(lambda s : s.split(':'), node.get('style', 'opacity:1.0').rstrip(';').split(';')))['opacity']);
```
Comment (kjellmf) [inline svg2tikz/extensions/tikz_export.py]: Is the commit message correct? You comment out opacity

File: svg2tikz/extensions/tikz_export.py [hunk 2]
```diff
@@ -1268,18 +1267,19 @@ def _write_tikz_path(self, pathdata, options=None, node=None):
...
             elif cmd == 'image':
                 closed_path = False;
-                pic += "\\node[anchor=north west,inner sep=0, scale=\globalscale] (image) at (%s,%s) {\includegraphics[width=%spt,height=%spt]{%s}}" % params;
+                pic += "(image) at (%s,%s) {\includegraphics[width=%spt,height=%spt]{%s}}" % params;
+#                 pic += "\\node[anchor=north west,inner sep=0, scale=\globalscale] (image) at (%s,%s) {\includegraphics[width=%spt,height=%spt]{%s}}" % params;
 #                 pic += "\draw (%s,%s) node[below right]  {\includegraphics[width=%spt,height=%spt]{%s}}" % params;
 			
         if options:
-            optionscode = "[%s]" % ','.join(options)
+            optionscode = "[%s]" % ','.join(options);
```
Comment (kjellmf) [inline svg2tikz/extensions/tikz_export.py]: Please remove semicolon

File: svg2tikz/extensions/tikz_export.py [hunk 3]
```diff
@@ -1351,7 +1351,7 @@ def _output_group(self, group, accumulated_state=None):
                 logging.debug("Unhandled element %s", node.tag)
 
             goptions, transformation = self.convert_svgstate_to_tikz(graphics_state, accumulated_state, node)
-            options = transformation + goptions + options
+            options = transformation + goptions + options;
```
Comment (kjellmf) [inline svg2tikz/extensions/tikz_export.py]: Semicolon

**Comment by kjellmf** [comment]
There are quite a few merge conflicts. Could you please resolve them? If you rebase from master and apply your changes most of them should be resolved.

File: svg2tikz/extensions/tikz_export.py (hunk 1 above)
Comment (aalavandhaann) [reply svg2tikz/extensions/tikz_export.py]: We can safely ignore this and yes the commit message is wrong. I will remove this line in the future PRs.

**Comment by kjellmf** [comment]
Closing this while waiting for a revised pull request

**Pull request closed without merging**